### DISCUSSION TITLE
Installing Geocore in host site.

### DISCUSSION THREAD

**Discussion by Delp1960** [geodesicsolutions-community/geocore-community#180] [Q&A]
Hi all! Great to find this forum.
I'm not real tech savvy, but I have decided to install the Geocore software in a sub-domain to have a play with it.
I've followed the Community Edition Installation Instructions up to Step 7.
"Run the web-based installation script located at setup/index.php using your favorite Internet browser. For example, if your site was example.com, you would run example.com/setup/index.php in your browser. It will guide you through all the steps."

       I type this into my browser & it goes nowhere.  I've uploaded the Geocore files into my hosting site. 
I guess I'm missing something here.  If I open setup/index.php locally with Chrome i just get the first page of the Geocore Wizard.

Any suggestions would be appreciated!

Thanks.

**Comment by jonyo** [comment]
When you say `I type this into my browser & it goes nowhere.` what exactly does it do?  White screen, error page, "not found" page or what?  Or if you are able, providing a screenshot might help.  Thanks!

**Reply by Delp1960** [reply]
![classifiedstimeout](https://user-images.githubusercontent.com/108928579/178086319-7fe3c015-8eb3-47b6-8608-3243f444ba6c.JPG)

**Reply by Delp1960** [reply]
I don't get how one could access a website by simply typing that command.

**Reply by hektyc** [reply]
@Delp1960 from the look of your screenshot it appears that your website is not setup correctly meaning that your DNS at your domain registrar is not pointing to your server.

**Reply by vicos59** [reply]
https://griffithclassifieds.com.au works now, but there is nothing there except a subdir:

https://griffithclassifieds.com.au/griffithclassifieds.com.au/

**Comment by mascu1605** [comment]
Hello, i have a problem with a new install

Hostet on Strato

1. Install via Geodesic Solutions Community Wiki - Software Installation
2. install Ioncubeloader 

after install can not go to the website or adminpanel

![geocore](https://user-images.githubusercontent.com/109048387/178168980-9baa6c91-b22d-4372-82af-ed89a23d3f04.jpg)

![geocore](https://user-images.githubusercontent.com/109048387/178172690-b7f46094-0ba1-4839-bf3f-f957322b3e40.jpg)

Sorry for my english i´m german

**Reply by vicos59** [reply]
Which version of PHP are you using?

**Comment by mascu1605** [comment]
![geocore](https://user-images.githubusercontent.com/109048387/178378337-30c45581-971c-4d8c-a7a3-9b5717d902fb.jpg)

**Reply by jonyo** [reply]
If you are installing GeoCore Comunity Edition you do not need ioncube anymore for the main software.  We may need to update the wiki pages to make it clear if it still talks about ioncube somewhere.
